tasks/section1.yml

```diff
@@ -1272,15 +1277,20 @@ print $NF }'"
       - rule_1.7.1.5
 
 - name: "SCORED | 1.7.1.5 | PATCH | Ensure permissions on /etc/issue are configured"
-  command: /bin/true
+  file:
+      dest=/etc/issue
+      state=file
+      owner=root
+      group=root
+      mode=0644
```

Comment:
This task needs to be in YAML literal form per https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md